src/projects/pull-request/pull-request-page.html

```diff
@@ -114,6 +113,11 @@
         margin: 0 auto;
         width: 75%;
       }
+      iron-pages[wide] {
```

Comment:
I think we need a different solution for this `wide` mode. At this point, when I click the 'Show Space' button, the entire pull-request-view goes into wide-mode, making the comments and description section way to wide. I want the wide-mode to only be active if the space view is visible.